Source/VisualRecognitionV3/VisualRecognition.swift

```diff
@@ -254,13 +250,15 @@ public class VisualRecognition {
        recommended pixel density is 32X32 pixels per inch, and the maximum image size is 10 MB. Redirects are followed,
        so you can use a shortened URL.
        You can also include images with the **images_file** parameter.
+     - parameter acceptLanguage: The desired language of parts of the response. See the response for details.
      - parameter imagesFileContentType: The content type of imagesFile.
      - parameter headers: A dictionary of request headers to be sent with this request.
      - parameter completionHandler: A function executed when the request completes with a successful result or error
      */
     public func detectFaces(
         imagesFile: URL? = nil,
         url: String? = nil,
+        acceptLanguage: String? = nil,
         imagesFileContentType: String? = nil,
```

Comment:
I guess adding an optional param is breaking anyway since the `headers` are at the end. The other SDKs have been making a handwritten change to add `acceptLanguage` after `imagesFileContentType`.